tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "tslint-no-unused-expression-chai"
```

Comment:
What is this about?

Reply:
A certain part of chai syntax is being flagged by tslint in error. The npm module readme has more info:

https://www.npmjs.com/package/tslint-no-unused-expression-chai

Reply:
Could you throw up a PR on resin-lint for this, chai is something we use in a load of places so having this rule available by default probably makes sense - at the very least it'll raise up a discussion around how we should handle it

Reply:
https://github.com/balena-io/node-resin-lint/pull/37 PR is up on lint